ansible_collections/arista/avd/extensions/molecule/eos_designs_negative_unit_tests/molecule.yml

```diff
@@ -1,4 +1,16 @@
 ---
+ansible:
+  executor:
+    backend: ansible-playbook
+    args:
+      ansible_playbook:
+        - --inventory=inventory/hosts.yml
+  env:
+    ANSIBLE_FORCE_COLOR: true
+    PYTHONPATH: ${PYTHONPATH}:${MOLECULE_SCENARIO_DIRECTORY}/custom_modules
```

Comment:
Note to self: do we still need this (maybe the answer is yes :) ) for eos_negative_unit_tests?